zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/PermissionsServiceProxy.java

```diff
@@ -0,0 +1,302 @@
+package com.github.onsdigital.zebedee.permissions.service;
+
+import com.github.onsdigital.zebedee.exceptions.BadRequestException;
+import com.github.onsdigital.zebedee.exceptions.NotFoundException;
+import com.github.onsdigital.zebedee.exceptions.UnauthorizedException;
+import com.github.onsdigital.zebedee.exceptions.ZebedeeException;
+import com.github.onsdigital.zebedee.json.CollectionDescription;
+import com.github.onsdigital.zebedee.json.PermissionDefinition;
+import com.github.onsdigital.zebedee.model.Collection;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.teams.model.Team;
+import com.github.onsdigital.zebedee.user.model.User;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * A proxy class that sits in front of the 2 permissions service implementations (legacy & JWT). The class does not
+ * perform any permisisons logic it determines which instance to invoke (legacy or JWT) based on the value of the
+ * feature flag.
+ */
+public class PermissionsServiceProxy implements PermissionsService {
+
+    private final PermissionsService legacyPermissionsService;
+    private final PermissionsService jwtPermissionsService;
+    private final boolean jwtSessionsEnabled;
+
+    public PermissionsServiceProxy(boolean jwtSessionsEnabled,
+                                   PermissionsService legacyPermissionsService,
+                                   PermissionsService jwtPermissionsService) {
+
+        this.legacyPermissionsService = legacyPermissionsService;
+        this.jwtPermissionsService = jwtPermissionsService;
+        this.jwtSessionsEnabled = jwtSessionsEnabled;
+    }
+
+    /**
+     * @param session {@link Session} to get the user details from.
+     * @return
+     * @throws IOException
+     */
+    @Override
+    public boolean isPublisher(Session session) throws IOException {
+        if (jwtSessionsEnabled) return jwtPermissionsService.isPublisher(session);
```

Comment:
Can we update this to include `{`after the `if` statement please - Im not a fan of this style and we should be consistent with the rest of the codebase.

```
if (jwtSessionsEnabled) {
    return jwtPermissionsService.isPublisher(session);
}
return legacyPermissionsService.isPublisher(session);
```